connectors/exercism/populate.py

```diff
@@ -0,0 +1,79 @@
+from git import Repo
+from os import listdir, walk
+from models import User, Exercise, Artifact
+from uuid import uuid1
+
+
+def get_or_create_default_user(session):
+    default_user = session.query(User).filter(
+        User.email == 'hello@exercism.io').first()
+    if not default_user:
+        exercism_user = User(email="hello@exercism.io",
+                             name="exercism.io",
+                             nickname="exercism",
+                             is_teacher=True,
+                             github_token="")
+        session.add(exercism_user)
+        return exercism_user
+    return default_user
+
+
+def upload_folder(exercise_path, starting_path, blob_destination, bucket):
+    for dirpath, dirname, filenames in walk(exercise_path):
+        rel_dir = dirpath.split(starting_path)[1]
+        for file in filenames:
+            blob = bucket.blob(f"{blob_destination}/{rel_dir}/{file}")
+            blob.upload_from_filename(f"{dirpath}/{file}")
+
+
+def fetch_exercise(name, starting_path, blob_prefix, mapper, bucket, user):
+    print(f"Fetching exercise:{name}")
+    exercise_path = f"{starting_path}/{name}"
+    upload_folder(exercise_path,
+                  starting_path,
+                  mapper.language,
+                  bucket)
+
+    has_hint = mapper.hint_exists(exercise_path)
+    mappings = mapper.get_files_mappings(blob_prefix,
+                                         name,
+                                         has_hint=has_hint)
+    exercise = Exercise(name=name,
+                        language=mapper.language,
+                        description=mapper.pluck_readme(exercise_path),
+                        creator=user.email)
+
+    files = Artifact(readme=mappings["readme"],
+                     solution=mappings["solution"],
+                     starting_point=mappings["starting_point"],
+                     test=mappings["test"],
+                     hint=mappings["hint"])
+    return (exercise, files)
```

Comment:
With the relationship properly set, you can add the exercise as a child of files through the relationship:
```python
    files = Artifact(readme=mappings["readme"],
                     solution=mappings["solution"],
                     starting_point=mappings["starting_point"],
                     test=mappings["test"],
                     hint=mappings["hint"],
                     exercise=exercise)
   return files